itest/lnd_misc_test.go

```diff
@@ -230,6 +230,49 @@ func testSphinxReplayPersistence(ht *lntest.HarnessTest) {
 	ht.ForceCloseChannel(carol, chanPoint)
 }
 
+// testFundingExpiryBlocksOnPending checks that after an OpenChannel, and
+// before the funding transaction is confirmed, that the FundingExpiryBlocks
+// field of a PendingChannels decreases.
+func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
+	// Configure bob to require 5 confs for a channel to be considered open
```

Comment:
nit: missing fulls top at end of sentence.

Reply:
Also, this comment just describes what the code does. But it doesn't say _why_ the default confs need to be overwritten (because the default in all integration tests is set to 1).
We should probably also decrease the value to 3 since the more blocks we mine here, the slower subsequent tests become because the nodes need to sync to the extra blocks mined.

Reply:
@guggero 

> We should probably also decrease the value to 3 since the more blocks we mine here, the slower subsequent tests become because the nodes need to sync to the extra blocks mined.

Is the test not isolated in this instance? If so, would it make sense to restart the node with the defaults when the test ends (both on success and failure)?

Side note: Do tests need to do their own cleanup here? I did a quick search for test setup, but I didn't see how this was handled.

Reply:
there is a cleanup in `Subtest`. And nope, currently all tests in the same tranche share the same blockchain state, which is something I'm looking into for a change.